src/targets/mobile/actions/rating.js

```diff
@@ -0,0 +1,28 @@
+/* global __APP_VERSION__ */
```

Comment:
If you use redux, you should create a single file with everything for the redux state:

```js
// constant may be used only in this file
const ADD = 'my-app/users/ADD';

// default export is the reducer
export default function reducer(state = {}, action = {}) {
  switch (action.type) {
    ADD: return {...state, [action.payload.id]: action.payload}
    default: return state;
  }
}

// Action Creators
export function addUser(payload) {
  return { type: ADD, payload };
}

// Selectors
export function getUsers (state) {
 return state
}

export function getUser (state, id) {
  return state[id]
}
```